applications/crossbar/doc/ref/system_configs.md

```diff
@@ -46,6 +46,16 @@ curl -v -X POST \
     http://{SERVER}:8000/v2/system_configs/{SYSTEM_CONFIG_ID}
 ```
 
+#### Patch
```

Comment:
Please migrate these changes to `crossbar/doc/system_config.md` and flesh them out appropriately